_topic_maps/_topic_map.yml

```diff
@@ -2551,6 +2547,17 @@ Topics:
     File: cluster-logging-collector
   - Name: Collecting and storing Kubernetes events
     File: cluster-logging-eventrouter
+- Name: Log storage
+  Dir: log_storage
+  Topics:
+  - Name: About log storage
+    File: log-storage-overview
+  - Name: Installing log storage
+    File: installing-log-storage
+  - Name: Logging using LokiStack
+    File: cluster-logging-loki
```

Comment:
Is it possible to rename this section to `Configuring the LokiStack log store` or something like this?

Reply:
Good call, can do